examples/nature_of_code/chp_06_agents/6_01_seek.rs

```diff
@@ -0,0 +1,159 @@
+// The Nature of Code
+// Daniel Shiffman
+// http://natureofcode.com
+
+// Seeking "vehicle" follows the mouse position
+
+// Implements Craig Reynold's autonomous steering behaviors
+// One vehicle "seeks"
+// See: http://www.red3d.com/cwr/
+extern crate nannou;
+
+use nannou::prelude::*;
+use nannou::Draw;
+
+fn main() {
+    nannou::app(model, event, view).run();
+}
+
+struct Model {
+    vehicle: Vehicle,
+}
+
+struct Vehicle {
+    position: Vector2,
```

Comment:
Using `Point2` might be a little clearer as "point"s should generally refer to a specific location within space, while "vector"s describe some sort of translation or direction in space.

This used to matter a lot more when we used `cgmath` for the `Point2` and `Vector2` types whose behaviour were quite different (they had different trait implementations and different inherent methods), but since we switched to using our own it no longer really matters as `Point2` is simply a type alias for `Vector2` and they should both implement everything.

Also, I guess Processing uses PVector for everything, so maybe it's nicer to keep things simpler anyways - I'm not really sure!

Up to you whether or not you want to change this - just thought I'd mention it so you have the backstory n Point2 vs Vector2 :)

Reply:
No worries, Might just leave this one as it's consistent with the example